charts/identity-trust-bundle/Chart.yaml

```diff
@@ -0,0 +1,36 @@
+# #############################################################################
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+
+apiVersion: v2
+name: identity-trust-bundle
+description: |
+  Identity & Trust Bundle Chart for Catena-X.
+  This bundle includes:
```

Comment:
This might not be the correct way to list bundle componentes as this might not be correct with BYO component and later toggling which component you wanna use. This might lead to irritation as description does not match real configuration.